### PR TITLE
MF-115:Location/facility selectors should default to the last used by user

### PR DESCRIPTION
**Pull request by donaldkibet**
MF-115
This PR resolves this ticket [MF-115](https://issues.openmrs.org/projects/MF/issues/MF-115?filter=allissuesl)

### CODE REVIEW

File: src/location-picker/location-picker.component.tsx [hunk 1]
```diff
@@ -55,8 +56,16 @@ interface LocationPickerProps {
...
 const LocationPicker: React.FC<LocationPickerProps> = (props) => {
+  const userDefaultLoginLocation: string = "userDefaultLoginLocationKey";
+  const getDefaultUserLoginLocation = (): string => {
+    return window.localStorage.getItem(
+      `${userDefaultLoginLocation}${props.currentUser}`
+    );
+  };
   const [locationData, setLocationData] = React.useState<LocationDataState>({
-    activeLocation: "",
+    activeLocation: getDefaultUserLoginLocation()
```
Comment (FlorianRappl) [inline src/location-picker/location-picker.component.tsx:66]: Rather use `getDefaultUserLoginLocation() ?? ''`.

On the other hand, this is a bit fishy as `localStorage` may get corrupted or the available locations may change. Thus we definitely will need to check if the location is valid.

**Review by FlorianRappl — COMMENTED** [review]
Looks good in general :beers:

Maybe we'd need to refine how the location is checked.

File: src/location-picker/location-picker.component.tsx (hunk 1 above)
Comment (donaldkibet) [reply src/location-picker/location-picker.component.tsx]: I have added a check for validation for the login locations.

**Review by FlorianRappl — APPROVED** [review]
LGTM :beers:!

**Pull request merged**